test/Users.js

```diff
@@ -95,3 +97,169 @@ describe("2360: Users settings", () => {
       });
   });
 });
+
+describe("2370: Change password", () => {
+  before(async () => {
+    const loginResponseIngestor = await utils.getIdAndToken(appUrl, {
+      username: "adminIngestor",
+      password: TestData.Accounts["adminIngestor"]["password"],
+    });
+    accessTokenAdminIngestor = loginResponseIngestor.token;
+
+    const loginResponseUser1 = await utils.getIdAndToken(appUrl, {
+      username: TestData.Accounts["user1"]["username"],
+      password: TestData.Accounts["user1"]["password"],
+    });
+    userIdUser1 = loginResponseUser1.userId;
+    accessTokenUser1 = loginResponseUser1.token;
+
+    const loginResponseUser2 = await utils.getIdAndToken(appUrl, {
+      username: TestData.Accounts["user2"]["username"],
+      password: TestData.Accounts["user2"]["password"],
+    });
+    userIdUser2 = loginResponseUser2.userId;
+    accessTokenUser2 = loginResponseUser2.token;
+
+    db.collection("User").updateOne(
+      { username: TestData.Accounts["user2"]["username"] },
+      { $set: { authStrategy: "oidc" } },
+    );
+  });
+
+  after(async () => {
+    db.collection("User").updateOne(
+      { username: TestData.Accounts["user2"]["username"] },
+      { $set: { authStrategy: "local" } },
```

Comment:
Shouldn't `authStrategy` set to `oidc`?

Reply:
authStrategy is set to oidc for validation testing. After the validation testing, the user data should revert to the original one to avoid any potential issues.